ha/setup/ha_setup.py

```diff
@@ -374,9 +378,7 @@ def process(self):
                         self._add_node(node, cluster_user, cluster_secret)
             else:
                 # Add node with SSH
-                for node in nodelist:
-                    if node != node_name:
-                        self._add_node_remotely(node, cluster_user, cluster_secret)
+                self._add_node_remotely(node_name, cluster_user, cluster_secret)
```

Comment:
I guess this change is not related to this ticket too, some branch update issue.

Reply:
No this change it got some merge problem

Can you check now patch is updated properly